content/special-topics/program-invocation/darktable-cli.md

```diff
@@ -67,9 +74,30 @@ _where `<FORMAT>` is the name of the selected output format.__
 `--style-overwrite`
 : The specified style overwrites the history stack instead of being appended to it.
 
+`--apply-custom-presets <0|1|false|true>`
+: Whether to load `data.db` which contains presets and styles. Disabling this option allows to run multiple instances of `darktable-cli` at the cost of inability to use `--style` option. Defaults to true.
+
+`--out-ext <extension>`
+: Set the output extension to use. If specified takes precedence over `<output file>`. Defaults to extracted from `<output file>` or `jpg` in case `<output folder>` is specified.
+
+`--import <file or dir>`
+: Specify input file or folder, can be used multiple times. Do not combine this option with `<input file or folder>`.
```

Comment:
What happens if you try to? Does it fall in a heap? If so, may be better to say "This option cannot be combined with..."